src/VerifyResourceUsage.pl

```diff
@@ -1,13 +1,13 @@
-#!/usr/bin/perl
 ######################################################################
-# verify_resource_usage.pl
+# VerifyResourceUsage.pl
```

Comment:
Do you think there is value in calling VerifyResourceUsage.pl from our build.ps1 script?
It would be nice to have these checks as part of our build script, imo.

Reply:
I created an issue for this: https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/issues/1094